data/beaconrestapi/src/integration-test/java/tech/pegasys/teku/beaconrestapi/beacon/PostBlindedAndUnblindedBlockTest.java

```diff
@@ -96,21 +104,22 @@ void preDeneb(
 
     prepareResponse(request, version);
 
-    postRequestAndAssert(
-        route,
-        request,
-        signedBeaconBlockSchema,
-        Optional.of(SpecMilestone.BELLATRIX.name()),
-        useSsz,
-        version);
+    final Optional<String> consensusHeader =
+        setConsensusHeader ? Optional.of(SpecMilestone.BELLATRIX.name()) : Optional.empty();
+
```

Comment:
shouldn't we remove the duplicated call left on line 111? since this is testing multiple combination including with ConsensusHeader set and not set?

Reply:
Good catch, you're right! Removed it